include/openssl/sha.h

```diff
@@ -205,7 +205,6 @@ OPENSSL_EXPORT int SHA384_Final(uint8_t out[SHA384_DIGEST_LENGTH],
 OPENSSL_EXPORT uint8_t *SHA384(const uint8_t *data, size_t len,
                                uint8_t out[SHA384_DIGEST_LENGTH]);
 
-
 // SHA-512.
```

Comment:
This is probably what's throwing off the pre-push checker
```suggestion

// SHA-512.
```